listado-de-libros/scss/detail.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import 'equalizer.scss';
+@import 'colors.scss';
+
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
+.nav-bar {
+  height: 70px;
+  width: 1169px;
+  background-color: $white;
+  box-shadow: 0 4px 10px 0 $black-shadow;
+  position: sticky;
+  top: 0;
+}
+
+.logo-wolox {
+  height: 70px;
+  width: 212.42px;
+  margin-left: 120px;
+}
+
+.all-group {
+  height: 768px;
+  width: 1169px;
+  background-color: $wild-sand;
+}
+
+.all-group2 {
+  display: flex;
+  align-content: center;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.group {
+  height: 300px;
+  width: 198px;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-shadow;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 36px;
+  margin-right: 36px;
```

Comment:
Always use the `margin` property instead of specifying in multiple margin properties. For example, this can be:

`margin: 35px 35px 0 0;`